modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -0,0 +1,251 @@
++
+[source,terminal]
+----
+$ oc apply -f _<my-catalog-source.yaml>_ <1>
+----
+<1> Replace _<my-catalog-source.yaml>_ with your catalog source configuration file name.
+
+. To verify that the catalog source is applied, check for the `READY` state by using the following command:
++
+[source,terminal]
+----
+$ oc describe catalogsources.operators.coreos.com -n openshift-marketplace operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+Name:         operatorhubio-catalog
+Namespace:    openshift-marketplace
+Labels:       <none>
+Annotations:  <none>
+API Version:  operators.coreos.com/v1alpha1
+Kind:         CatalogSource
+Metadata:
+  Creation Timestamp:  2024-01-31T10:09:46Z
+  Generation:          1
+  Resource Version:    2811
+  UID:                 60ce4a36-86d3-4921-b9fc-84d67c28df48
+Spec:
+  Display Name:  Community Operators
+  Grpc Pod Config:
+    Security Context Config:  restricted
+  Image:                      quay.io/operatorhubio/catalog:latest
+  Publisher:                  OperatorHub.io
+  Source Type:                grpc
+  Update Strategy:
+    Registry Poll:
+      Interval:  60m
+Status:
+  Connection State:
+    Address:              operatorhubio-catalog.openshift-marketplace.svc:50051
+    Last Connect:         2024-01-31T10:10:04Z
+    Last Observed State:  READY <1>
+  Registry Service:
+    Created At:         2024-01-31T10:09:46Z
+    Port:               50051
+    Protocol:           grpc
+    Service Name:       operatorhubio-catalog
+    Service Namespace:  openshift-marketplace
+Events:                 <none>
+----
+<1> The status is reported as `READY`.
+
+. Confirm that the catalog source is running by using the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-marketplace -l olm.catalogSource=operatorhubio-catalog
+----
++
+.Example output
+[source,terminal]
+----
+NAME                          READY   STATUS    RESTARTS   AGE
+operatorhubio-catalog-j7sc8   1/1     Running   0          43s
+----
+
+. Create a Subscription CR configuration file by using the following example YAML:
++
+.Example Subscription custom resource YAML
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: my-gitlab-operator-kubernetes
+  namespace: olm-microshift <1>
+spec:
+  channel: stable
+  name: gitlab-operator-kubernetes
+  source: operatorhubio-catalog
+  sourceNamespace: openshift-marketplace <2>
+----
+<1> The specific namespace. Operators reference the global namespace for content, but run in the `olm-microshift` namespace.
+<2> The global namespace. Subscriptions CRs in any namespace can reference catalogs created in the `openshift-marketplace` namespace.
+
+. Apply the Subscription CR configuration by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f _<my-subscription-cr.yaml>_
+----
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/my-cert-manager created
```

Comment:
This is not consistent with the name used in the spec

Reply:
This is my output. I think it uses the name under metadata instead of name under spec

`oc apply -f my_customresource.yaml
subscription.operators.coreos.com/my-gitlab-operator-kubernetes created`